core-services/prow/02_config/generate-boskos.py

```diff
@@ -73,6 +73,12 @@
     'aws-perf-qe-quota-slice': {
         'us-west-2': 3,
     },
+    'aws-perfscale-qe-quota-slice': {
+        'us-west-2': 3,
```

Comment:
@yunjiang29 This quota will be too small for us. As we have several Self Managed, Classic Rosa and Hypershift cron jobs and also multiple team members actively working on adding more tests and analyzing results/debugging etc.. 

Can we set this to 'us-west-2': 15 at least?

Reply:
@svetsa-rh  This number means the **concurrent** job running in the CI. Even for aws-qe profile, 40 is enough for all of our tests.
I'd suggest to set to 5, and see if it works. WDYT?

Reply:
@yunjiang29 Even for aws-qe I have hit lease issues and quota issues several times last 2-3 weeks. `'aws-perfscale-qe-quota-slice'` is the new profile we will be using for majority of tests.
I think 5 will be little too less. May be we can start with 10 and see if we hit any issues.

Reply:
@svetsa-rh you are using `aws-qe` for your testing, and you will switch to `aws-perfscale-qe-quota-slice` after it's ready., is it right?

Reply:
correct @yunjiang29 - for Classic Rosa and HCP, we will switch right away.

Reply:
correct @yunjiang29 - for Classic Rosa and HCP, we will switch right away.